packages/config/src/ConfigService.ts

```diff
@@ -52,7 +51,7 @@ let configuration: configOpts = {
 
 function checkAddress(): void {
   if (!configuration.address) {
-    throw ERROR_WS_ADDRESS_NOT_SET()
+    throw Error('Node address to connect to not configured!')
```

Comment:
I suggest we merge this after the utils, then you can import the error from there